### PR TITLE
Don't send default quality with Imgix auto compression

### PR DESCRIPTION
**Pull request by jorenvanhee**
This PR changes the behaviour for the configuration `imgixConfig.default.defaultParams`.

**Current situation**

```
'defaultParams' => [
    'auto' => 'compress',
],
```

The `auto=compress` and `q=80` parameters are sent to Imgix. Even though we didn't set the `q` parameter. So it is currently impossible to only use `auto=compress`.

**Pull request**

```
'defaultParams' => [
    'auto' => 'compress',
],
```

Only the `auto=compress` parameter is sent to Imgix. You let Imgix decide on the quality `q`. According to the [Imgix docs](https://docs.imgix.com/apis/url/auto/auto#compress), the quality will be set to `45` if `auto=compress` is used.

```
'defaultParams' => [
    'auto' => 'compress',
    'q' => '80',
],
```

You can still set `q` if you want to make the decision yourself.

---

So basically. This PR makes it possible to let Imgix decide on the `q` quality parameter if you're using `auto=compress`.

### CODE REVIEW

**Comment by aelvan** [comment]
This has been merged. Thanks for contributing!
